proxy/proxy.go

```diff
@@ -882,7 +883,22 @@ func (p *Proxy) makeBackendRequest(ctx *context) (*http.Response, *proxyError) {
 
 	p.metrics.IncCounter("outgoing." + req.Proto)
 	ctx.proxySpan.LogKV("http_roundtrip", StartEvent)
-	response, err := p.roundTripper.RoundTrip(req)
+
+	var response *http.Response
+	switch req.URL.Scheme {
+	case "fastcgi":
+		rt, err := NewFastCgiRoundTripper(p.log, req.URL.Host)
+		if err != nil {
+			p.log.Errorf("could not create fastcgi roundtripper: %v", err)
```

Comment:
In case you log, start with uppercase letter, in case you return error you start with lowercase.
I suggest:
`"Failed to create fastcgi roundtripper: %v"`

Reply:
Fixed